examples/awaits.rb

```diff
@@ -38,47 +46,46 @@
       false
     end
   end
-
-  # Let the user know we're  ready and listening..
-  event.respond 'Guess a number between 1 and 10..'
+  event.respond "My number was: `#{magic}`."
 end
 
-# Above we used the provided User#await method to easily set up
+# Above we used the provided User#await! method to easily set up
 # an await for a follow-up message from a user.
 # We can also manually register an await for specific kinds of events.
 # Here, we'll write a command that shows the current time and allows
 # the user to delete the message with a reaction.
-# We'll be using Bot#add_await to do this:
-# https://www.rubydoc.info/gems/discordrb/Discordrb%2FBot:add_await
+# We'll be using Bot#add_await! to do this:
+# https://rubydoc.info/github/swarley/discordrb/Discordrb%2FBot:add_await!
```

Comment:
I would prefer it if the documentation references did not point to my repo for the time being. I can see about updating it repo wide in a separate PR later.

Reply:
Do you want this to point to the drb git repo or drb gem? I'd vote for the git repo since it's a bit more up to date.

Reply:
```suggestion
# https://www.rubydoc.info/gems/discordrb/Discordrb%2FBot:add_await!
```